dataplane/api-redbeams/client/databases/databases_client.go

```diff
@@ -25,95 +25,157 @@ type Client struct {
 }
 
 /*
-DeleteDatabase deletes database config by name
+DeleteDatabaseByCrn deletes a database config by c r n
```

Comment:
NIT c r n -> CRN, here and below

Reply:
Yeah, that's Swagger for you. It's auto-generated text, I suppose because the source says CRN in all caps.